src/test/crimson/seastore/test_object_data_handler.cc

```diff
@@ -800,22 +800,9 @@ TEST_P(object_data_handler_test_t, overwrite_then_read_within_transaction) {
       auto pin1 = remap_pin(*t, std::move(pins.front()), 4096, 8192);
       auto ext = get_extent(*t, base + 4096, 4096 * 2);
       ASSERT_TRUE(ext->is_exist_clean());
-      ext = tm->get_mutable_extent(*t, ext)->cast<ObjectDataBlock>();
-
-      auto l = 4096;
-      memset(
-	known_contents.c_str() + base + 4096,
-	'z',
-	l);
-      bufferlist bl;
-      bl.append(
-	bufferptr(
-	  known_contents,
-	  base + 4096,
-	  l));
-
-      ext->overwrite(0, bl);
+      write(*t, base + 4096, 4096, 'y');
```

Comment:
OK, object_data_handler can do overwrite to the exist-clean extent after the change.